student.rb

```diff
@@ -0,0 +1,17 @@
+module StudentModule
+  require_relative 'person'
+  include PersonModule
+
+  class Student < Person
+    attr_accessor :classroom
```

Comment:
- [ ] Please make the `attr_accessor :classroom` abode the class student. so it will be accessible before starting the definition.